### PR TITLE
DP-29104 eliminate lightest font weight in the system

### PR DESCRIPTION
**Pull request by clairesunstudio**
Changed:
- Removed the lightest font-weight for text in the system
- Space out font-weight scale evenly to better show visual hierarchy, bump up all text font weights by at least 50
- Consolidate hard-coded font weights into variable
- Remove styling overrides on telephone links, so they appear as other contact links 

Before:
```scss
$fonts-lightest: 150 !default; // only used for styling non text font weights, e.g. separator |
$fonts-lighter:  250 !default; (removed)
$fonts-light:    350 !default; // body font weight
$fonts-normal:   400 !default; // (bumped up to 450)
$fonts-bold:     550 !default; 
$fonts-bolder:   650 !default;
```

After:
```scss
$fonts-lightest: 150 !default; // only used for styling non text font weights, e.g. separator |
$fonts-light:    400 !default; // body font weight
$fonts-normal:   500 !default;
$fonts-semibold: 550 !default;
$fonts-bold:     600 !default;
$fonts-bolder:   700 !default;
```

Before:
Screen Shot 2023-08-24 at 11 01 50 AM

After:
Screen Shot 2023-08-24 at 4 38 47 PM

### CODE REVIEW

**Comment by ygannett** [comment]
@clairesunstudio Thank you for your work.

For the review, I used the [table below](https://www.w3.org/WAI/GL/WCAG3/2020/methods/font-characteristics-contrast/) for reference.  This is a part of WCAG 3.0 support resources.  It's still under development, but WCAG 2.1 doesn't have anything solid to evaluate font weights for their appropriation as reasonable readability.  I hope this gives us some sort of baseline to work with.
apca_contrast-lookup-table

I also want to point out that Noto font is generally thinner than the reference fonts W3C uses (Helvetica Neue, Helvetica, Fira Sans, Kanit, and Arial) as you see below:
sample-fontweight
It seems Noto's font-weight is 100 lighter than those reference fonts.  For example, to match the font-weight of 350 of those fonts visually, it seems Noto's font-weight needs to be 450 as I played around.  We probably need to set the base font-weight heavier for text.  This is something to think about when WCAG 3.0 is released around 2026.  I'm open to discussion.

With the current base font-weight of 350, the following components appear to need a heavier weight:
1. ma__contact-group__details
     - patterns/05-pages-bio-page/05-pages-bio-page.html
     bio-oage_ma__contact-group__details

     - patterns/05-pages-contact-row-examples/05-pages-contact-row-examples.html
     contact-row_ma__contact-group__details

1. ma__figure__caption
     - patterns/05-pages-information-details/05-pages-information-details.html
     figure-caption

1. ma__image-credit__container
     - patterns/05-pages-information-details/05-pages-information-details.html
     image-creadit

1. ma__action-step__header span
     - patterns/05-pages-location-park-content/05-pages-location-park-content.html
alert-updatetimestamp

1. secondary-label-close span
     - patterns/05-pages-policy-advisory-directive/05-pages-policy-advisory-directive.html
close-symbol

**Comment by clairesunstudio** [comment]
@ygannett thanks for your thorough review - I fixed the font weight for `x` used as the close icon and adjusted to visually match the others in the system. 

For the rest of the problem areas, based on your research, I think it makes sense to systematically bump up the values by 50, so the lightest font weight variable value in Mayflower for text doesn't fail accessibility. Here's what I changed --> https://github.com/massgov/mayflower/pull/1841/files. This makes all the fonts slightly thicker which will benefit accessibility overall and address the problematic components. Not guaranteed to meet 3.0 yet, but a step closer for sure.

**Review by ygannett — APPROVED** [review]
Thank you for the changes.  No need to meet the 3.0 guideline yet.  That is something to keep in mind as an up-coming change.  That's why I wanted to mention that.  Its approach is more practical for users who are affected by readability issues.  So, accommodating the approach would help those users for sure.

The changes look good.  Once the backstop references are updated, ready to merge.

File: changelogs/DP-29104.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Changed:
+  - project: Assets
+    component: Variables
+    description: Remove the lightest font-weight for text in the system. Space out font-weight scale evenly to better show visual hierarchy.  Consolidate hard-coded font weights into a new variable. (#1831)
```
Comment (clairesunstudio) [inline changelogs/DP-29104.yml]:
```suggestion
    description: Remove the lightest font-weight for text in the system. Space out font-weight scale evenly to better show visual hierarchy.  Consolidate hard-coded font weights into a new variable. (#1831)
```
```suggestion
    description: Remove the lightest font-weight for text in the system. Space out font-weight scale evenly to better show visual hierarchy, bump up all text font weights by at least 50.  Consolidate hard-coded font weights into a new variable. (#1831)
```

**Pull request merged**